src/index.js

```diff
@@ -1,11 +1,20 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './App';
+import App from './components/App';
+import 'bootstrap/dist/css/bootstrap.css';
+import * as actions from './actions/book';
+import store from './reducers/index';
+// import configureStoreApp from './reducers/configureStore';
+
+// const store = configureStoreApp();
+
+store.dispatch(actions.bookAdded('book 1', 'description of book 1', 'Horror'));
+store.dispatch(actions.bookAdded('book 2', 'description of book 2', 'Action'));
+store.dispatch(actions.bookAdded('book 3', 'description of book 3', 'Kids'));
 
 ReactDOM.render(
   <React.StrictMode>
     <App />
   </React.StrictMode>,
```

Comment:
- [ ] <b>OPTIONAL:</b> None of you components have access to the store whatsoever. since this is a new change, It wasn't pointed out by the previous reviewer. so I will mark it as `optional` But I strongly recommend to make sure that your components have access to the store by wrapping `<App />` in Provider.
```Javascript
 <Provider store={store}>
    <App />
 </Provider>

```